### PR TITLE
adding recipe for RemovedJakartaFacesExpressionLanguageClasses jakarta ee 10

### PR DESCRIPTION
**Pull request by ranuradh**
## What's changed?
image

## Anyone you would like to review specifically?
@joanvr , @timtebeek 

### Checklist
- [x] I've added unit tests to cover both positive and negative cases
- [x] I've added the license header to any new files through `./gradlew licenseFormat`
- [x] I've used the IntelliJ IDEA auto-formatter on affected files

### CODE REVIEW

File: src/test/java/org/openrewrite/java/migrate/jakarta/RemovedJakartaFacesExpressionLanguageClassesTest.java [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+import org.openrewrite.test.RecipeSpec;
+import org.openrewrite.test.RewriteTest;
+
+import static org.openrewrite.java.Assertions.java;
+
+public class RemovedJakartaFacesExpressionLanguageClassesTest implements RewriteTest {
+
+    @Override
+    public void defaults(RecipeSpec spec) {
+        spec.parser(JavaParser.fromJavaVersion().classpathFromResources(new InMemoryExecutionContext(), "jakarta.el-api-4.0.0","jakarta.faces-2.3.19", "jakarta.faces-3.0.3")).recipe(Environment.builder().scanRuntimeClasspath("org.openrewrite.java.migrate.jakarta").build().activateRecipes("org.openrewrite.java.migrate.jakarta.RemovedJakartaFacesExpressionLanguageClasses"));
```
Comment (joanvr) [inline src/test/java/org/openrewrite/java/migrate/jakarta/RemovedJakartaFacesExpressionLanguageClassesTest.java:31]: Could you format this a bit? Similar to here: https://github.com/openrewrite/rewrite-migrate-java/blob/cc6668dd741dedb00a79b8b40718fc9306d5936b/src/test/java/org/openrewrite/java/migrate/jakarta/RemovedIsParmetersProvidedMethodTest.java#L32-L36

Comment (joanvr) [inline src/test/java/org/openrewrite/java/migrate/jakarta/RemovedJakartaFacesExpressionLanguageClassesTest.java:31]: is jakarta.el-api-4.0.0 also needed? What types come from each jar?

Comment (ranuradh) [reply src/test/java/org/openrewrite/java/migrate/jakarta/RemovedJakartaFacesExpressionLanguageClassesTest.java]: updated in next commit

Comment (ranuradh) [reply src/test/java/org/openrewrite/java/migrate/jakarta/RemovedJakartaFacesExpressionLanguageClassesTest.java]: Without that my tests were failing hence added it. 
jakarta.faces.el.*  comes from jakarta.faces-3.0.3
javax.faces.el.* comes from jakarta.faces-2.3.19
jakarta.el.* comes from jakarta.el-api-4.0.0

**Review by joanvr — APPROVED** [review]

**Comment by joanvr** [comment]
Thank you again for your continuous contributions @ranuradh ! :)

**Pull request merged**